composer.json

```diff
@@ -23,33 +23,35 @@
         }
     ],
     "require": {
-        "php": "^5.3.3|^7.0",
-        "doctrine/common": "^2.3|^3.0",
-        "symfony/config": "^2.3|^3.0",
-        "symfony/http-kernel": "^2.3|^3.0",
-        "symfony/http-foundation": "^2.4|^3.0",
-        "symfony/dependency-injection": "^2.3|^3.0"
+        "php": "^5.4|^7.0",
+        "doctrine/common": "^2.5",
+        "symfony/config": "^2.8|^3.0",
```

Comment:
I not kicking out, but encouraging to update to 2.8
Everyone can go to 2.8 without pain, just upgrading. People still can use 2.3 with this bundle, but they will not be able to use new developments.
The problem would be by supporting only 3.0, but is not the case.
Of course, would be better to maintain all versions, but then maintenance of the project turns a mess.